src/frontend/components/actions/edit.tsx

```diff
@@ -41,15 +41,20 @@ const Edit: FC<ActionProps> = (props) => {
     return false
   }
 
+  const cssTag = `${resource.id}-${action.name}-drawer-content`
```

Comment:
I would unify naming a bit.

`DrawerContent`'s `data-css` is named:
`User-edit-drawer-content` in this file but for bulk delete it's just
`User-bulk-delete-content`.

I think the best way would be to create:
`src/frontend/utils/css-tags.ts`

with a set of functions, for example:

```
export const getActionElementCss = (resource: ResourceJSON, action: ActionJSON, suffix: string) => `${resource.id}-${action.name}-${suffix}`
```

Usage:
```
const drawerContentTag = getActionElementCss(resource, action, 'drawer-content')
```

Reply:
done